src/Compilers/VisualBasic/Portable/Binding/Binder_Attributes.vb

```diff
@@ -794,7 +794,8 @@
                                    Not _binder.IsValidTypeForAttributeArgument(conv.Operand.Type) Then
 
                                     If Not conv.HasErrors Then
-                                        ReportDiagnostic(diagBag, conv.Operand.Syntax, ERRID.ERR_RequiredAttributeConstConversion2, conv.Operand.Type, conv.Type)
+                                        ' BC30934: Conversion from '{0}' to '{1}' cannot occur in a constant expression used as an argument to an attribute.
+                                        ReportDiagnostic(diagBag, conv.Operand.Syntax, ERRID.ERR_RequiredAttributeConstConversion2, If(conv.Operand.Type, _binder.Compilation.GetSpecialType(SpecialType.System_Object), conv.Type)
```

Comment:
>) [](http://example.com/codeflow?start=214&length=1)

Missing ```)```?

Reply:
Yup sorry. I edited directly on GitHub

```suggestion
                                        ReportDiagnostic(diagBag, conv.Operand.Syntax, ERRID.ERR_RequiredAttributeConstConversion2, If(conv.Operand.Type, _binder.Compilation.GetSpecialType(SpecialType.System_Object), conv.Type))
```

Reply:
I think the ```)``` was added in the wrong place

Reply:
Yes :(

Sorry again. Fixed